pl/README.md

```diff
@@ -11,17 +11,21 @@ ogromną ilość swojego wolnego czasu tłumacząc ponad 80% całego kursu. Doda
 
 ## Wprowadzenie
 
-Czy kiedykolwiek czułaś, że wokół jest coraz więcej nowych technologii, a Ty zostajesz w tyle? Zastanawiałaś się kiedyś jak zrobić stronę internetową, ale nie miałaś motywacji by zacząć? Czy zdarzyło Ci się pomyśleć, że świat programowania jest dla Ciebie zbyt skomplikowany by nawet spróbować zrobić coś samej?
+Czy chciałabyś nauczyć się czarować? Z perspektywy osoby niemagicznej rzecz wydaje się dość złożona. Nie każdy wie, jak zacząć. Nie jest też łatwo znaleźć sklep z różdżkami. Jak ich używać? Na samym początku nie wiadomo nawet kogo i o co pytać. Nie każdy dostaje bilet wstępu do specjalistycznych szkół, gdzie sporą cześć wąptliwości rozwiewa kontakt z profesorami. Przez to sztuka czarowania uchodzi za dziedzinę tylko dla wybranych. Podobnie jest z programowaniem, czyli sztuką czarowania przy użyciu komputera. Komputer sam w sobie nie jest przedmiotem magicznym. Wiele osób niemagicznych posiada nawet jego wersje zminiaturyzowane z dotykowym interfejsem. Programowanie sprawia, że urządzenie słucha Twoich rozkazów - wtedy dzieje się magia. Czy zastanawiałaś się, czym są programy, które nim sterują? Czy postawiłaś sobie kiedyś za cel napisanie takiego programu? Co chciałabyś, żeby robił? Zdajemy sobie sprawę, że początki bywają trudne, a część  internetowych serwisów, gdzie mogłaś szukać pomocy, zasiedlają kosmici (nie kwestionujemy jednak ich wiedzy). Po takim starciu łatwo się zniechęcić, dojść do wniosku "to nie dla mnie, nie nadaję się do tego". A  co, jeśli to nieprawda? Zdradzimy Ci pewien sekret: programowanie nie jest takie trudne. Pokażemy Ci, że może być niezłą zabawą.
```

Comment:
Nie wiem co zrobić z tym akapitem, bo wydaje mi się zawiły i trudny do przeczytania, oraz zdaje mi się, że zawiera sporo słów nieszczególnie przyjaznych początkującym - zminiaturyzowana wersja komputera, you know :)
Potrzebuję, żeby się ktoś inny wypowiedział tutaj, bo nie umiem ocenić :P